wordpress-mobile/gutenberg-mobile
# Fix change list type crash

This PR updates the Gutenberg ref to test the [`Fix change list type crash` PR](https://github.com/WordPress/gutenberg/pull/15010). 

<img width="389" alt="Screen Shot 2019-04-17 at 10 22 34 AM" src="https://user-images.githubusercontent.com/9772967/56272628-6b041c00-60fb-11e9-932c-7a07d928d504.png">

**To test:**
- Run the app.
- Add a ListBlock.
- Change the list type to Ordered List.
- Check that it doesn't crash.
